include/aspect/simulator_access.h

```diff
@@ -534,6 +553,13 @@ namespace aspect
       get_lateral_averaging () const;
 
       /**
+       * Return a pointer to the object that describes the
+       * current constraints.
+       */
+      const ConstraintMatrix &
+      get_current_constraints() const;
```

Comment:
The term `constraints` is so overloaded. I know that `current_constraints` is the name of the variable we use, but maybe we rename the function to `get_current_dof_constraints()`? What do you guys think?

Also, maybe add to the documentation that "current" refers to the "time step we are solving at the moment"?